spec/models/campagne_spec.rb

```diff
@@ -25,4 +25,14 @@
       it { expect(campagne.code).to eq '12345' }
     end
   end
+
+  describe '#evaluation' do
+    let(:campagne) { create :campagne }
+    let!(:evaluation1) { create :evaluation, campagne: campagne }
```

Comment:
Personnellement je n'aime pas qu'on utilise les factories dans les tests de modèles, que je considère comme étant unitaires.
En général si je ressens le besoin de faire un test qui fait appel à la base, je préfère le mettre dans `spec/integration`.

En l'occurence ici, si tu veux tester de manière unitaire, tu peux utiliser te reposer sur shoulda-matchers pour écrire :

`it { should have_many(:evaluations).order('created_at desc')`

Je serais heureux d'avoir un échange ensemble si besoin de s'aligner sur la stratégie de tests :)

Reply:
Ok, je me range à la façon de faire générale